registry/remote/referrers.go

```diff
@@ -68,6 +68,38 @@ var (
 	errNoReferrerUpdate = errors.New("no referrer update")
 )
 
+const (
+	// opDeleteReferrerIndex represents the operation for deleting a
+	// referrer index.
+	opDeleteReferrerIndex = "DeleteReferrersIndex"
```

Comment:
nit: The variable name does not match the message. Is it `ReferrerIndex` or `ReferrersIndex`?